client/src/auth/sign-up.tsx

```diff
@@ -35,14 +43,12 @@ export default function SignUpApp() {
     const errors = JSON.parse(searchParams.get("errors") || "{}");
     return (
       <div>
-        <h2>Sign-up errors</h2>
-        <p>An error occurred trying to sign you up.</p>
+        <h2>Sign-in errors</h2>
+        <p>An error occurred trying to complete sign in.</p>
         <pre>{JSON.stringify(errors, null, 2)}</pre>
         <p>
-          <Link to={`/${locale}/signin`}>
-            Try starting over the sign-in process
-          </Link>
-          .
+          We apologize for the inconvenience,{" "}
+          <Link to={`/${locale}/signin`}>please try signing-in again</Link>.
```

Comment:
This should be 2 sentences now. I.e.
```
We apologize for the inconvenience. <a>Please try signing in again</a>.
```
Also note that "signing in" is a verb whereas "signing-in" is a noun.